finals-2024-01-16/code/5042-ivan-apostolovski/src/app/models/student.ts

```diff
@@ -1,4 +1,20 @@
-export interface Student {
+export interface Bands {
```

Comment:
The model names should be
- singular (`Band` not `Bands`)
- not in a file called `student.ts`

+4 points for the models, the definitions are ok.